.github/workflows/ci.yaml

```diff
@@ -42,4 +42,64 @@ jobs:
           run: npm i --include=dev
 
         - name: Lint JavaScript
-          run: npm run lint
\ No newline at end of file
+          run: npm run lint
+
+  tests:
+    name: Running unit tests
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_DB: hedera
+          POSTGRES_USER: hedera
+          POSTGRES_PASSWORD: hedera
+        # Set health checks to wait until postgres has started
+        options: >-
```

Comment:
Cool! Didn't know there was an option for health checks.

Reply:
Nor I! This is cribbed from https://docs.github.com/en/actions/using-containerized-services/about-service-containers -- they have examples for postgres and redis. :)